examples/platform/linux/CommissionerMain.cpp

```diff
@@ -105,7 +105,7 @@ class MyServerStorageDelegate : public PersistentStorageDelegate
     }
 };
 
-class MyCommissionerCallback : public CommissionerCallback
```

Comment:
Please comment as to why this "custom" callback set must be implemented. It's to support specific policies that are not the default of the likes of chip-tool. The reasons for this should be made clear

Reply:
Added separate issue to make this configurable: https://github.com/project-chip/connectedhomeip/issues/22459